### PR TITLE
Fix cleanvm merging to non-Staging

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

**Comment by rhafer** [comment]
Looks good to me. Though I am not really able to give a ±1 here as I lack some more detailed jenkins knowledge.

File: scripts/jenkins/jobs-obs/openstack-submit.yml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            Devel:Cloud:2.0)
+                submitcmd="$OSC copypac -K -e"
+                OSCBUILDDIST=SLE_11_SP3
+            ;;
+            Cloud:OpenStack:[EF]*)
+                submitcmd="$OSC copypac -K -e"
+            ;;
+            Cloud:OpenStack:[GHI]*)
+                submitcmd="$OSC copypac -K -e"
+                OSCBUILDDIST=SLE_11_SP3
```
Comment (vuntz) [inline scripts/jenkins/jobs-obs/openstack-submit.yml]: There's nothing for [JKL]?

**Comment by vuntz** [comment]
Looks like the project is not sent to the triggered job: https://ci.opensuse.org/view/OpenStack/job/openstack-submit/3/

File: scripts/jenkins/jobs-obs/openstack-submit.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    node: cloud-trackupstream
+
+    parameters:
+      - string:
+          name: openstack_project
+          default: None
+          description: name of the main project that should be updated with packages from its staging project
+
+    builders:
+      - update-automation
```
Comment (vuntz) [inline scripts/jenkins/jobs-obs/openstack-submit.yml]: Do we really need this bit?

**Comment by vuntz** [comment]
+1

**Comment by toabctl** [comment]
+1

**Pull request merged**